install-dependencies.sh

```diff
@@ -293,6 +294,7 @@ elif [ "$ID" = "fedora" ]; then
     pip3 install "geomet<0.3,>=0.1"
     # Disable C extensions
     pip3 install scylla-driver --install-option="--no-murmur3" --install-option="--no-libev" --install-option="--no-cython"
+    pip3 install traceback-with-variables
```

Comment:
Please check that the relocatable package correctly finds the module. I don't know if it looks for pip modules when it packages python modules.

Reply:
@avikivity I tested the module worked on deb/rpm packaged relocatable scylla-python3, by adding "raise Exception" on scripts, it just worked.
Also I verified traceback-with-variables PIP package is added at /opt/scylladb/python3/lib64/python3.9/site-packages/.